src/main/java/net/neoforged/neoforge/fluids/FluidType.java

```diff
@@ -1071,5 +1113,31 @@ public Properties rarity(Rarity rarity) {
             this.rarity = rarity;
             return this;
         }
+
+        /**
+         * Allows this fluid to drip from Pointed Dripstones and fill cauldrons below.
```

Comment:
This should probably clarify that its specifically for dripstone _stalactites_. Other references should be modified accordingly. (Preference should be given to calling this phenomenon as 'stalactite dripping', in my opinion.)